azext_iot/central/commands_device.py

```diff
@@ -69,13 +69,29 @@ def registration_info(
     token=None,
     central_dns_suffix="azureiotcentral.com",
     device_status=None,
+    max_devices=None,
+    summarize_registration=False,
 ):
     provider = CentralDeviceProvider(cmd=cmd, app_id=app_id, token=token,)
     if not device_id:
```

Comment:
looks like the code under `if not device` is now longer than the other case. Let's reverse the logic in that case:

```
if device:
  return provider.get_device_registration_info()

# do multi-device stuff here
```
```